README.md

```diff
@@ -440,151 +517,230 @@ Before setting up the following Survey application, dependencies given below sho
+            curl -OJL https://raw.githubusercontent.com/ELEVATE-Project/samiksha- 
+            service/refs/heads/feature/sample_data_scripts/documentation/1.0.0/native/scripts/macos/create-databases.sh
             ```
-      2. Make the executable by running the following command:
+         2. Make the executable by running the following command:
 
             ```
             chmod +x create-databases.sh
             ```
-      3. Run the script file:
+         3. Run the script file:
 
             ```
             ./create-databases.sh
             ```
-   -  **MacOS**
+            
+      -  **Windows**
 
-      1. Download `create-databases.sh` Script File:
+         1. Download `create-databases.bat` Script File:
 
-         ```
-         curl -OJL https://raw.githubusercontent.com/ELEVATE-Project/samiksha- 
-         service/refs/heads/feature/sample_data_scripts/documentation/1.0.0/native/scripts/macos/create-databases.sh
-         ```
-      2. Make the executable by running the following command:
+            ```
+            curl -OJL https://raw.githubusercontent.com/ELEVATE-Project/samiksha- 
+            service/refs/heads/feature/sample_data_scripts/documentation/1.0.0/native/scripts/windows/create-databases.bat
+            ```
+         2. Run the script file:
+
+            ```
+            create-databases.bat
+            ```
 
-         ```
-         chmod +x create-databases.sh
-         ```
-      3. Run the script file:
 
-         ```
-         ./create-databases.sh
-         ```
 
 6.  **Run Migrations To Create Tables**
 
-   -  **Ubuntu/Linux/MacOS**
+      -  **Ubuntu/Linux/MacOS**
 
-      1. Install Sequelize-cli globally:
+         1. Install Sequelize-cli globally:
 
-         ```
-         sudo npm i sequelize-cli -g
-         ```
-      2. Run Migrations:
+            ```
+            sudo npm i sequelize-cli -g
+            ```
+         2. Run Migrations:
 
-         ```
-         cd user/src && npx sequelize-cli db:migrate && cd ../.. && \
-         cd notification/src && npx sequelize-cli db:migrate && cd ../..
-         ```
+            ```
+            cd user/src && npx sequelize-cli db:migrate && cd ../.. && \
+            cd notification/src && npx sequelize-cli db:migrate && cd ../..
+            ```
+      -  **Windows**
+
+         1. Run Migrations:
+
+            ```
+            cd user\src && npx sequelize-cli db:migrate && cd ..\.. &&
+            cd notification\src && npx sequelize-cli db:migrate && cd ..\..
+            ```
 
 7.  **Enabling Citus And Setting Distribution Columns (Optional)**
 
       To boost performance and scalability, users can opt to enable the Citus extension. This transforms PostgreSQL into a distributed database, spreading data across multiple nodes to handle large datasets more efficiently as demand grows.
 
       > NOTE: Currently only available for Linux based operation systems.
```

Comment:
@MallanagoudaB You can remove the NOTE from here as you have added windows citus setup as well.